pkg/ccl/backupccl/backup_processor.go

```diff
@@ -355,7 +347,7 @@ func runBackupProcessor(
 						splitMidKey = true
 					}
 
-					req := &roachpb.ExportRequest{
+					KVReq := &roachpb.ExportRequest{
```

Comment:
we usually lead with lowercase for local vars, since otherwise they look like exported package vars, so e.g. `kvReq`.  

the name seems a tad weird to me, since the only "requests" we ever make are kv requests, and they're typically just called "req" throughout the codebase (e.g. `git  grep "req :=.*roachpb.*Request"`), but it is technically correct I suppose?